scripts/template-helm-chart.sh

```diff
@@ -266,3 +266,13 @@ done
 perl -i -0777 -pe 's/^/{{- if and .Values.alloy.enabled (not .Values.alloy.alloy.configMap.create) -}}\n/ if 1 .. 1' $TEMPLATE_DIR/verticadb-operator-alloy-cm.yaml
 perl -i -0777 -pe 's/name: \{\{ include "vdb-op.name" \. \}\}-alloy/name: vdb-op-alloy/' $TEMPLATE_DIR/verticadb-operator-alloy-cm.yaml
 echo "{{- end }}" >> $TEMPLATE_DIR/verticadb-operator-alloy-cm.yaml
+
+# 28. Conditionally create alloy rbac resources
+for f in $TEMPLATE_DIR/verticadb-operator-alloy-sa.yaml \
+  $TEMPLATE_DIR/verticadb-operator-alloy-cr.yaml \
+  $TEMPLATE_DIR/verticadb-operator-alloy-crb.yaml
+do
+  perl -i -0777 -pe 's/^/{{- if and .Values.alloy.enabled (not .Values.alloy.alloy.configMap.create) -}}\n/ if 1 .. 1' $f
+  perl -i -0777 -pe 's/name: \{\{ include "vdb-op.name" \. \}\}-alloy/name: vdb-op-alloy/g' $f
```

Comment:
```suggestion
  perl -i -0777 -pe 's/name: \{\{ include "vdb-op.name" \. \}\}-alloy/name: alloy-vertica-sa/g' $f
```
Also, only the servicaaccount must be changed. Only when the serviceaccount is set to this specific value do we want to generate the sa, role and rolebinding. If another value is passed, we assume the user has their own serviceaccount.